src/nft/components/details/detailsV2/TraitRow.tsx

```diff
@@ -27,21 +45,32 @@ const TraitRowValue = styled(ThemedText.BodySmall)<{ $flex?: number; $justifyCon
   justify-content: ${({ $justifyContent }) => $justifyContent};
 `
 
-export const TraitRow = ({ trait }: { trait: Trait }) => {
+export const TraitRow = ({ trait, collectionAddress }: { trait: Trait; collectionAddress: string }) => {
+  const [rowHovered, toggleRowHovered] = useReducer((s) => !s, false)
   // TODO: Replace with actual rarity, count, and floor price when BE supports
```

Comment:
NIT: I would add a jira ticket here for tracking

Reply:
added existing jira ticket to comment